src/membership/members.rs

```diff
@@ -122,14 +150,16 @@ decl_storage! {
         /// MemberId to assign to next member that is added to the registry
         pub NextMemberId get(next_member_id) build(|config: &GenesisConfig<T>| config.first_member_id): T::MemberId = T::MemberId::from(DEFAULT_FIRST_MEMBER_ID);
 
-        /// Mapping of member ids to their corresponding primary accountid
+        /// Mapping of member ids to their corresponding primary account_id
```

Comment:
Change the name of the storage field to be more direct, e.g. `PrimaryAccountByMemberId`.

Also, do we actually need two-way lookup MemberId <=> PrimaryAccount?

Reply:
It is primarily needed to store the keyspace of the member accounts map.
Otherwise there wouldn't be a way to iterate over all members.

The Actors module uses it to lookup the member account id of a storage provider to send rewards to.

To get rid of it totally we need to switch to using the `enumerable` version of the StorageMap. I think its called  linked_map.